android/app/src/main/kotlin/com/jabook/app/jabook/compose/data/preferences/ProtoSettingsRepository.kt

```
@@ -116,96 +188,226 @@ public interface SettingsRepository {
      */
-    public suspend fun clearSleepTimerState()
+    public suspend fun clearSleepTimerState() = Unit
 
     /**
      * Reset all settings to defaults.
      */
+    public suspend fun resetToDefaults() = Unit
+}
+
+/** Contract for application settings backed by Proto DataStore. */
+public interface SettingsRepository {
```

detekt (1.23.8)

[warning] The interface SettingsRepository is too complex. Consider splitting it up.

(detekt.complexity.ComplexInterface)